jf_agent/jira_download.py

```diff
@@ -40,11 +40,28 @@ def download_users(jira_connection):
 
 
 # Returns an array of Field dicts
-def download_fields(jira_connection):
+def download_fields(jira_connection, include_fields, exclude_fields):
+
     print('downloading jira fields... ', end='', flush=True)
-    result = jira_connection.fields()
+
+    filters = []
+    if include_fields:
+        filters.append(lambda field: field['key'] in include_fields)
+    if exclude_fields:
+        filters.append(lambda field: field['key'] not in exclude_fields)
+
+    fields = []
+    for field in jira_connection.fields():
+        keep = True
+        for filt in filters:
+            if not filt(field):
+                keep = False
+                break
+        if keep:
+            fields.append(field)
+
```

Comment:
I would have done that with a list comprehension:

`fields = [field for field in jira_connection.fields() if all(filt(field) for filt in filters)]`